index.html

```diff
@@ -0,0 +1,239 @@
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins:wght@200&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!--the nav-bar -->
+    <div id="the-nav-cont" class="the-general-cont">
+      <div class="navvv-cont">
+        <a href="#" class="logo">Mahan Sanjarani</a>
+        <nav class="navvbar">
+          <a href="#" class="opt">
+            <img src="./img/menu/Icon-Menu.png" alt="option icon" />
+          </a>
+          <a href="" class="n-b-list">Portfolio</a>
+          <a href="#about" class="n-b-list">About</a>
+          <a href="#contact" class="n-b-list">Contact</a>
+          <a href="" class="n-b-list-icon"
+            ><img src="./img/menu/Mail-icon.png" alt="Mail"
+          /></a>
+        </nav>
+      </div>
+    </div>
+
+    <!--the intro -->
+    <div class="main-cont the-general-cont">
+      <div class="sub-m-cont">
+        <h1 class="wcm-greeting">
+          Hey there. <br />
+          I'm Mahan
+        </h1>
+
+        <h2 class="prof">A Web Developer</h2>
+        <p class="welcome-msg">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+
+        <ul class="socials-icon-links">
+          <li>
+            <a href="https://github.com/mahanalyytech"
+              ><img
+                src="./img/github-logo/github.png"
+                alt="github-logo"
+                class="github"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/mahan-sanjarani-678358247 "
+              ><img
+                src="./img/linked-in-logo/linked-in-2.png"
+                alt="linked-in-logo"
+                class="linked-in"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img
+                src="./img/two-fingers-logo/two-fingre-logo.svg"
+                alt="two-fingers-logo"
+            /></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/MahanSanjarani"
+              ><img
+                src="./img/twitter-logo/twitter.png"
+                alt="twitter"
+                class="twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/m-logo/m-logo.svg" alt="m-logo" /></a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <!--my work-section -->
```

Comment:
- [ ] Please consider wrapping the work section (the heading and the cards) in a `section` element instead of a div, this is more semantically correct.